scopes/explorer/command-bar/ui/autocomplete-input/autocomplete-input.module.scss

```diff
@@ -8,6 +8,8 @@
   outline: none;
 
   &::placeholder {
-    color: var(--text-placeholder-color, #babec9);
+    // TODO - reenable when using @teambit/documenter.theme.theme-context > 4.0.1 or @teambit/base-ui.theme.theme-provider > 1.0.1
+    // color: var(--bit-text-inactive, #babec9);
+    color: #babec9;
```

Comment:
why not just let the fallback catch it if `--bit-text-inactive` isnt available?

Reply:
`--bit-text-inactive` is already included in the `themeProvider`, but has the incorrect value `$d30` (too dark).

I already tagged a new theme provider with the correct color (`$d20`), but I can't use this version because it already removed the old colors.
I will update when using the new theme.